Packs/Workday/Integrations/WorkdaySignOnEventCollector/WorkdaySignOnEventCollector.yml

```diff
@@ -0,0 +1,110 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: Workday Sign On Event Collector
+  version: -1
+configuration:
+- name: base_url
+  display: Server URL (e.g., https://services1.myworkday.com)
+  required: true
+  defaultvalue: 'https://services1.myworkday.com'
+  type: 0
+  additionalinfo: 'API endpoint of Workday server. Can be obtained from the View API Clients report in the Workday application.'
+  section: Connect
+- name: tenant_name
+  display: Tenant Name
+  required: true
+  defaultvalue:
+  type: 0
+  additionalinfo: 'The name of the Workday Tenant. Can be obtained from View API Clients report in Workday application.'
+  section: Connect
+- name: credentials
+  display: Username
+  required: true
+  defaultvalue:
+  type: 9
+  displaypassword: Password
+  section: Connect
+  hiddenusername: false
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  type: 8
+  additionalinfo:
+  section: Connect
+  advanced: true
+- name: proxy
+  display: Use system proxy settings
+  required: false
+  type: 8
+  additionalinfo:
+  section: Connect
+  advanced: true
+- additionalinfo: The maximum number of sign on events to retrieve. Large amount of events may cause performance issues.
+  defaultvalue: '10000'
```

Comment:
This is the number we can handle in a minute? Isn't it too big?

Reply:
Product wanted it increased to 10,000 on the demo. We should be okay to handle it.